cloudconfig/userdatacfg_test.go

```diff
@@ -725,6 +725,56 @@ func (*cloudinitSuite) TestCloudInitConfigure(c *gc.C) {
 	}
 }
 
+func (s *cloudinitSuite) TestCloudInitConfigCloudInitUserData(c *gc.C) {
```

Comment:
I think this test will fail on windows so you will need to c.Skip() and add a TODO to re-enable once windows support added

Reply:
it's not failing when i run all of the unit tests.  It appears that most userdatacfg test are with the windows version. There is one test only for Windows: TestWindowsCloudInit.